examples/create-react-app-typescript/test/upload-and-check-recording.js

```diff
@@ -0,0 +1,66 @@
+const assert = require("assert");
+const { WebSocket } = require("ws");
+const { SimpleProtocolClient } = require("@replayio/protocol");
+const replay = require("@replayio/replay");
+
+(async () => {
+  try {
+    const apiKey = process.env.REPLAY_API_KEY;
```

Comment:
nit: Worth `assert`ing there's an API key here and failing early?

Otherwise this value gets passed through a few layers of abstraction that try to read cached tokens or other environment variables and eventually throws, but it would be clearer in this case to throw here (since we know this is the only expected entrypoint for an api key)